src/lib/loginSchema.js

```diff
@@ -0,0 +1,10 @@
+import * as Yup from 'yup';
+
+const loginSchema = Yup.object().shape({
+  email: Yup.string()
+    .email('Por favor, insira um endereço de e-mail válido.')
+    .required('Por favor, preencha este campo.'),
+  password: Yup.string().required('Por favor, preencha este campo.'),
```

Comment:
Yes, the validation in the `loginSchema` is different from the `signupSchema`. However, I'm afraid I didn't quite understand the suggestion you made. Could you please provide me with more clarity.

Reply:
i was talking about the number of characteres validation in just 1 of the schemas

![image](https://user-images.githubusercontent.com/75763403/228852659-ca282b08-9678-40b2-9d19-6e20eac594e1.png)

Reply:
Initially, I had considered not including the character validation for the password field during the login process since the user would already have met the minimum password requirements during signup. However, upon consideration, I believe it would be beneficial to include this validation in the login process as well.